### PR TITLE
Get paragraph and list formatting in route check

### PR DESCRIPTION
**Pull request by dabreegster**
I found a different way to export to CSV that includes newlines and lists, so we can avoid having to manually recreate that formatting.

### CODE REVIEW

File: codegen/common.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Turn an exported Excel string into HTML, handling paragraphs and lists.
```
Comment (dabreegster) [inline codegen/common.py:1]: Now that everything's wrapped in a `<p>`, it looks weirder with the score icon on top:
image
We could use `<br />` instead? Kind of depends how we want to style this in #17

File: src/lib/FancyRadio.svelte [hunk 2]
```diff
@@ -30,7 +30,7 @@
           />
           <label class="govuk-label govuk-radios__label" for={id + thisValue}>
             <span style:background={thisColor}>{thisValue}</span>
-            {thisLabel}
+            {@html thisLabel}
```
Comment (dabreegster) [inline src/lib/FancyRadio.svelte:33]: Strings with HTML in each of the pages seems fine for this case. We could make a bunch of nested slots and embed HTML "properly", but that sounds hard for no clear gain

File: codegen/path_check.py
Comment (dabreegster) [inline codegen/path_check.py:1]: The py scripts could be deduped more, but I don't think it's useful -- each of them is slightly different in fields. It's very easy to edit one right now if needed

**Comment by dabreegster** [comment]
image
Lists!

File: codegen/common.py (hunk 1 above)
Comment (Pete-Y-CS) [reply codegen/common.py:1]: happy to merge it into my branch and work out how to make it work more nicely

Comment (dabreegster) [reply codegen/common.py:1]: Sounds good. We could change to use <br> there if it winds up being nicer

**Review by Pete-Y-CS — APPROVED** [review]

**Pull request merged**